### PR TITLE
Bugfix for problem mentioned in #273

### PR DESCRIPTION
**Pull request by alexdowad**

### CODE REVIEW

File: test/test_diagnostic.rb [hunk 1]
```diff
@@ -69,4 +69,28 @@ def test_multiline_render
...
+    @buffer.source = source
+
+    location = Parser::Source::Range.new(@buffer, 33, 34)
+    diag = Parser::Diagnostic.new(:error, :unexpected_token, { :token => 'tNL' },
+                                  location)
+
+    assert_equal([
+      '(string):2:32-3:0: error: unexpected token tNL',
+      '(string):2:   foo: ->() # I forgot my brace',
+      '(string):2:                                ',
```
Comment (whitequark) [inline test/test_diagnostic.rb]: There should really be a `^` under the newline.

Comment (alexdowad) [inline test/test_diagnostic.rb]: One character past the end of `brace`, in other words?

Makes sense... I don't have time to look into this right now (unfortunately), but when I do, I can fix it.

Comment (alexdowad) [inline test/test_diagnostic.rb]: A clarifying note: this is the actual output which is printed to the console right now by `ruby-parse`.

Comment (whitequark) [inline test/test_diagnostic.rb]: I'll do it myself.

**Comment by alexdowad** [comment]
Thanks @whitequark. By the way, I am wondering whether `"\n"` should really be considered to span 2 lines. (As well as `"some text\n"`.) If so, I think we should ensure that the multiline diagnostic format is not used in these situations.

**Comment by whitequark** [comment]
I did that.

**Pull request merged**